soc/arm/silabs_exx32/Kconfig

```diff
@@ -182,4 +193,12 @@ config SOC_GECKO_HAS_ERRATA_RTCC_E201
 	  feature enabled by setting CCV1TOP in RTCC_CTRL fails to wrap the counter
 	  when RTCC_CNT is equal to RTCC_CC1_CCV, as intended."
 
+config SOC_GECKO_HAS_HFRCO_FREQ
+	bool
+	help
+	  If enabled, indicates that configuration of HFRCO frequency is supported for this SOC.
```

Comment:
As is, the comment is misleading. The efm32hg, efm32wg series also support configuration of HFRCO frequency, however use a different method. What about saying:

If enabled, indicates that configuration of HFRCO frequency for this SOC is supported via FREQRANGE field.

If we add in the future support for  efm32hg, efm32wg series we would need to use another Kconfig option. This one should be reserved for SoCs that rely on FREQRANGE field. We could also rename it to `SOC_GECKO_HAS_HFRCO_FREQRANGE`.

Reply:
Done.